### PR TITLE
Update deploy tutorial

### PR DESCRIPTION
**Pull request by emnul**
Fixes a typo, a code log issue, and updates images

### CODE REVIEW

**Review by pasevin — APPROVED** [review]
Thanks! The only thing I would do is run the images through an optimizer. It can shave it down by up to 80% sometimes!

**Comment by emnul** [comment]
> Thanks! The only thing I would do is run the images through an optimizer. It can shave it down by up to 80% sometimes!

Do you have a web tool or software package recommendation for this? I've never done this before @pasevin

File: docs/modules/ROOT/pages/tutorial/deploy.adoc [hunk 1]
```diff
@@ -61,10 +61,6 @@ image::tutorial-deploy-upgrade-wizard.png[Deploy block upgrade wizard]
 +
 image::tutorial-deploy-end-wizard.png[Deploy block end wizard]
 
-. Copy the wallet address of your relayer through the icon, and send some Goerli ETH to it. Although this relayer will send the deploy transaction, you will execute the upgrade transaction through your wallet, so leave some Goerli ETH in your wallet.
```
Comment (cairoeth) [inline docs/modules/ROOT/pages/tutorial/deploy.adoc:64]: removing this because because we are assuming auto funding for testnet relayers?

Comment (emnul) [reply docs/modules/ROOT/pages/tutorial/deploy.adoc:64]: Yes, but I recognize that may not always be possible. What are your thoughts?

Comment (cairoeth) [reply docs/modules/ROOT/pages/tutorial/deploy.adoc:64]: not possible if chain is not goerli or no funds available -- should be fine for now.

**Review by cairoeth — APPROVED** [review]

**Pull request merged**